conan/cli/commands/test.py

```diff
@@ -17,7 +17,7 @@ def test(conan_api, parser, *args):
     """
     parser.add_argument("path", action=OnceArgument,
                         help="Path to a test_package folder containing a conanfile.py",
-                        default=".", nargs='?')
+                        default="test_package", nargs='?')
```

Comment:
I'd like to see the accompanying test to this.
I think this might be challenging, because the ``conan test <path> <ref>`` 2 positional parameters, if the first one is defaulted, then assuming the only argument is the second one? But if you provide 1 positional, it will be assigned to ``path`` and that would be confusing to undo?

Reply:
I've simplified the test, but it's already tested in https://github.com/conan-io/conan/pull/18964/files#diff-b7168c5345dd1bb08453bd311bbaae11b13d595a29b6dd963764143c1ccc9370R29

As far as I can tell, it's able to understand that if `path` is `nargs=?` (and has a default), and the other positional is required, then if the user only supplies one, it assumes that it's meant to go to the required, non-defaulted one.

I've also tested this in the `conan run` command, and there it also works.

`conan run "command --version"` is understood as `path="."`, command="command --version"`